charts/enterprise-kyverno-operator/templates/cr-custompolicysets.yaml

```diff
@@ -0,0 +1,21 @@
+{{- range .Values.policies.customPolicySetCharts }}
+---
+apiVersion: security.nirmata.io/v1alpha1
+kind: PolicySet
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/name: policyset
+    app.kubernetes.io/instance: {{ .name }}
+    app.kubernetes.io/part-of: nirmata-kyverno-operator
+  name: {{ .name }}
+  namespace: {{ template "enterprise-kyverno.namespace" $ }}
+spec:
+  source:
+    helm:
+      chartRepo: {{ .chartRepo }}
+      chartName: {{ .chartName }}
+      version: {{ .version }}
+      username: {{ .username }}
+      passwordSecret: {{ .passwordSecret }}
```

Comment:
I think we shouldkeep the kyverno namespace fixed here, and allow changing the password secret under that: 

      passwordSecret: "{{ include "enterprise-kyverno.namespace" $ }}/{{ .passwordSecret }}

Likely that if the password is in a secret in another namespace, and we don't have perms to read it, say.